api/__init__.py

```diff
@@ -119,7 +117,11 @@ def before_req():
         def after_req(response):
             stats.timing('api.request', (time.time()-request._start_time)*1000)
             return response
-
+            
+# https://github.com/FAForever/api/issues/40
+@app.before_request
+def before_request():
+    faf.db.init_db(app.config)
```

Comment:
I am not sure and have no idea ...
This was the only fix I come up with ...
Maybe another expert should investiage some time here ...